content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -20,257 +17,259 @@ typically requires a VPC or a VPN, as well as a container network that
+    `servicesIpv4Cidr`. Replace $MY_ZONE and $MY_PROJECT in the following example commands with the appropriate values to
+    obtain the CIDR after installation:
 
     {{< text bash >}}
-    $ export GCP_OPTS="--zone MY_ZONE --project MY_PROJECT"
-    $ @install/tools/setupMeshEx.sh@ generateClusterEnv MY_CLUSTER_NAME
+
+    $ ISTIO_SERVICE_CIDR=$(gcloud container clusters describe $K8S_CLUSTER --zone $MY_ZONE --project $MY_PROJECT --format "value(servicesIpv4Cidr)")
+    $ echo -e "ISTIO_CP_AUTH=MUTUAL_TLS\nISTIO_SERVICE_CIDR=$ISTIO_SERVICE_CIDR\n" > cluster.env
+
     {{< /text >}}
 
-    Here's an example generated file
+4.    Check the generate files is similar to the following example:
 
     {{< text bash >}}
+
     $ cat cluster.env
-    ISTIO_SERVICE_CIDR=10.63.240.0/20
+    ISTIO_CP_AUTH=MUTUAL_TLS
+    ISTIO_SERVICE_CIDR=10.55.240.0/20
+
     {{< /text >}}
 
-*   Generate DNS configuration file to be used in the VMs. This will allow apps on the VM to resolve
-cluster service names, which will be intercepted by the sidecar and forwarded.
+5.    If the VM only calls services in the mesh, you can skip this step. Otherwise, add the ports the VM exposes to
+    the cluster.env file with the following command. You can change the ports later.
 
     {{< text bash >}}
-    $ @install/tools/setupMeshEx.sh@ generateDnsmasq
+
+    $ echo "ISTIO_INBOUND_PORTS=3306,8080" >> cluster.env
+
     {{< /text >}}
 
-    Here's an example generated file
+6.    Extract the initial keys for the service account to use on the VMs.
+
+    {{<text bash>}}
+
+    $ kubectl -n $SERVICE_NAMESPACE get secret istio.default  \
+        -o jsonpath='{.data.root-cert\.pem}' |base64 --decode > root-cert.pem
+    $ kubectl -n $SERVICE_NAMESPACE get secret istio.default  \
+        -o jsonpath='{.data.key\.pem}' |base64 --decode > key.pem
+    $ kubectl -n $SERVICE_NAMESPACE get secret istio.default  \
+          -o jsonpath='{.data.cert-chain\.pem}' |base64 --decode > cert-chain.pem
 
-    {{< text bash >}}
-    $ cat kubedns
-    server=/svc.cluster.local/10.150.0.7
-    address=/istio-mixer/10.150.0.8
-    address=/istio-pilot/10.150.0.6
-    address=/istio-citadel/10.150.0.9
-    address=/istio-mixer.istio-system/10.150.0.8
-    address=/istio-pilot.istio-system/10.150.0.6
-    address=/istio-citadel.istio-system/10.150.0.9
     {{< /text >}}
 
 ### Setting up the machines
 
-As an example, you can use the following "all inclusive" script to copy
-and install the setup:
+1.    Install the debian package:
+
+    {{< text bash >}}
+
+    $ curl -L https://storage.googleapis.com/istio-release/releases/1.0.0/deb/istio-sidecar.deb > istio-sidecar.deb
+    $ dpkg -i istio-sidecar.deb
 
-{{< text bash >}}
-$ export GCP_OPTS="--zone MY_ZONE --project MY_PROJECT"
-$ export SERVICE_NAMESPACE=vm
-{{< /text >}}
+    {{< /text >}}
 
-If you are running on a GCE VM, run
+2.    Copy the cluster.env and the *.pem files to the VM.
```

Comment:
Should be 1. Copy the cluster.env and the *pem files to the VM.  Manually numbering does not typically yield correct results and is prone to error.  The markdown generator will do that work for you.